internal/scheduler/cleaner/cleaner_job.go

```diff
@@ -0,0 +1,55 @@
+package cleaner
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/boundary/internal/scheduler"
+)
+
+type cleanerJob struct {
+	w db.Writer
+}
+
+func newCleanerJob(w db.Writer) *cleanerJob {
+	return &cleanerJob{
+		w: w,
+	}
+}
+
+// Status reports the job’s current status. We never change these values as
+// this job never finishes.
```

Comment:
Not sure I follow the `this job never finishes` comment. Though I am fine with not reporting status here

Reply:
I removed this part of the comment, it was just confusing.